LCLSGeneral/LCLSGeneral/DUTs/DUT_EPS.TcDUT

```diff
@@ -0,0 +1,40 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4024.12">
+  <DUT Name="DUT_EPS" Id="{0134451e-57c2-478a-9365-5d6570cca7b2}">
+    <Declaration><![CDATA[TYPE DUT_EPS :
+STRUCT
+    // Contains EPS flags
+    {attribute 'pytmc' := '
+        pv: EPS:nFlags
```

Comment:
For the motion lib, this results in somewhat redundant PV names:
```
IM1L0:XTES:MMS:PLC:stEPSForwardEnable:EPS:bEPS_OK
```
Are there other situations where it's useful to have the `EPS` text included in the PV name like this?